src/app/core/services/ess-upload-file.service.ts

```diff
@@ -12,11 +12,14 @@ export class EssUploadFileService {
   private maxEncSelectionLimit: number;
   private showInfoMessage = false;
   private notifySingleEnc: Subject<boolean> = new Subject<boolean>();
+  private avgSizeofENC: number;
+  private estimatedENCSize: number;
```

Comment:
Please rename to `estimatedTotalSize`
```suggestion
  private estimatedTotalSize: number;
```

Reply:
resolved